grammar/stmt.g

```diff
@@ -0,0 +1,7 @@
+Stmt =
```

Comment:
It's also included as an expression, though?

Reply:
Hm, yea, assuming expression is fixed to deal with `;`.  I'm maybe a little confused because `parse_stmt_without_recovery` explicitly handles macros and does not defer to item or expr to handle it.  (i.e., why does `StmtKind` have a separate `Mac` variant?).